src/ert_storage/ext/graphene_sqlalchemy.py

```diff
@@ -30,6 +30,25 @@ class SQLAlchemyObjectType(_SQLAlchemyObjectType):
     class Meta:
         abstract = True
 
+    @classmethod
+    def __init_subclass_with_meta__(
+        cls,
+        interfaces: Iterable[Type[Interface]] = (),
+        resolver: Callable = None,
+        arguments: Dict[str, "Argument"] = None,
+        _meta: Optional[ObjectTypeOptions] = None,
+        **kwargs: Any,
+    ) -> None:
+        super(SQLAlchemyObjectType, cls).__init_subclass_with_meta__(
+            interfaces=interfaces, _meta=_meta, **kwargs
+        )
+
+        # Replace 'metadata_' field with 'metadata'
+        fields = cls._meta.fields
+        if "metadata_" in cls._meta.fields:
+            fields["metadata"] = fields["metadata_"]
+            del fields["metadata_"]
+
     def resolve_id(self, info: ResolveInfo) -> str:
```

Comment:
Not part of  your pr but i noticed this - info is not used - should it be there?

Reply:
I forgot to remove it. 👍